.github/workflows/main.yml

```diff
@@ -216,7 +216,7 @@ jobs:
       - name: clippy
         uses: clechasseur/rs-clippy-check@v3
         with:
-          args: --verbose --locked --features net,no-boards,external-interrupts,spi,storage -p riot-rs -p riot-rs-arch -p riot-rs-boards -p riot-rs-chips -p riot-rs-debug -p riot-rs-embassy -p riot-rs-embassy-common -p riot-rs-identity -p riot-rs-macros -p riot-rs-random -p riot-rs-rt -p riot-rs-storage -p riot-rs-threads -p riot-rs-utils -- --deny warnings
+          args: --verbose --locked --features csprng,net,no-boards,external-interrupts,spi,storage -p riot-rs -p riot-rs-arch -p riot-rs-boards -p riot-rs-chips -p riot-rs-coap -p riot-rs-debug -p riot-rs-embassy -p riot-rs-embassy-common -p riot-rs-identity -p riot-rs-macros -p riot-rs-random -p riot-rs-rt -p riot-rs-storage -p riot-rs-threads -p riot-rs-utils -- --deny warnings
```

Comment:
This was intentional: Without, builds would fail.

Which, in retrospect, means that riot-rs-coap should depend on the feature through Cargo. Reverted this addition; the Cargo feature dependency from riot-rs-coap to riot-rs-random/csprng will now satisfy that.

Reply:
Turns out I even had that dependency, but at the wrong level (from riot-rs/coap to riot-rs/csprng rather than from riot-rs-coap to riot-rs-random/csprng, which even works when riot-rs/coap is enabled). The last fixup e06bfb1a09b022d1cd64a083b0e303caff3b1101 removes that old dependency, which is redundant with the new one in dbeafd09ff036218ced4991d3c49e8aaa773cce7.